deploy/tasks/main.yaml

```
@@ -85,6 +91,7 @@
     path: '{{ deploy_helper.project_path }}'
     release: '{{ deploy_helper.new_release }}'
     state: finalize
+    keep_releases: '{{ keep_releases | default(5) }}'
```

```suggestion
    keep_releases: '{{ keep_releases }}'
```

The default is already set in `defaults/main.yaml`, no need to add it here to.